Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/command.yml

```diff
@@ -272,11 +272,11 @@
         points: 15
       - id: RMCAttachmentS42xTelescopicMiniscope
         points: 15
-    # - name: Helmet Visors # TODO RMC14 Port Helmet Welding Visor
-    #   entries:
-    #   - id: RMCHelmetWeldingVisor
-    #     points: 5
-    #     recommended: true
+    - name: Helmet Visors 
+      entries:
+        - id: RMCVisorWelding
+          points: 5
+          recommended: true
```

Comment:
The visor is intended to be down in other supplies and not recommended now, the duplicate flashlight is fine to remove though